README.md

```diff
@@ -21,26 +21,28 @@ can use one of the many wrappers available on npm, like
 
 ## Table of Contents
 
-- [Protocol support](#protocol-support)
-- [Installing](#installing)
-  - [Opt-in for performance and spec compliance](#opt-in-for-performance-and-spec-compliance)
-- [API docs](#api-docs)
-- [WebSocket compression](#websocket-compression)
-- [Usage examples](#usage-examples)
-  - [Sending and receiving text data](#sending-and-receiving-text-data)
-  - [Sending binary data](#sending-binary-data)
-  - [Simple server](#simple-server)
-  - [External HTTP/S server](#external-https-server)
-  - [Multiple servers sharing a single HTTP/S server](#multiple-servers-sharing-a-single-https-server)
-  - [Server broadcast](#server-broadcast)
-  - [echo.websocket.org demo](#echowebsocketorg-demo)
-  - [Other examples](#other-examples)
-- [FAQ](#faq)
-  - [How to get the IP address of the client?](#how-to-get-the-ip-address-of-the-client)
-  - [How to detect and close broken connections?](#how-to-detect-and-close-broken-connections)
-  - [How to connect via a proxy?](#how-to-connect-via-a-proxy)
-- [Changelog](#changelog)
-- [License](#license)
+- [ws: a Node.js WebSocket library](#ws-a-nodejs-websocket-library)
+  - [Table of Contents](#table-of-contents)
```

Comment:
I'm not sure if it makes sense to add the table of contents itself in the table of contents. Anyway no strong opinion.

Reply:
> I'm not sure if it makes sense to add the table of contents itself in the table of contents. Anyway no strong opinion.

Apologies for my blunder! :sob: 
Had a run-in with my formatter applying that automatically without me noticing.

I believe it's restored now after the commit I made.

Thanks for catching that! :+1:

Reply:
Seems good can you please apply the two suggested changes above? The numbered list is not rendered correctly anyway. Other than that LGTM.

Reply:
> Seems good can you please apply the two suggested changes above? The numbered list is not rendered correctly anyway. Other than that LGTM.

Should be good now.
The suggestion violated a code style rule, which I addressed